src/payment/PaymentForm.test.jsx

```diff
@@ -5,6 +5,7 @@ import configureMockStore from 'redux-mock-store';
 import { mount } from 'enzyme';
 import { SubmissionError } from 'redux-form';
 import { IntlProvider, configure as configureI18n } from '@edx/frontend-i18n';
+import * as analytics from '@edx/frontend-analytics';
```

Comment:
If you aren't actually using analytics and just want to mock it. I'd suggest not importing the package and instead stub it out.
`const analytics = { sendTrackEvent: jest.fn() }`

Reply:
Used `jest.mock('@edx/frontend-analytics', () => ({
  sendTrackEvent: jest.fn(),
}));`